package.json

```diff
@@ -29,11 +29,12 @@
   "scripts": {
     "lint": "npm run prettier && npm run eslint",
     "eslint": "eslint-godaddy --max-warnings 0 --ignore-pattern /coverage/ ./",
-    "prettier": "prettier --write lib",
+    "prettier": "prettier --write lib bin",
     "release": "standard-version -t ''",
     "test": "tap test/**/*.test.js",
     "start": "node lib/app.js",
-    "dev": "nodemon lib/app.js"
+    "dev": "LOCALSTACK=1 AWS_ACCESS_KEY_ID=foobar AWS_SECRET_ACCESS_KEY=foobar nodemon lib/app.js",
+    "localstack": "docker run -it -p 4569:4569 -p 9999:8080 -e SERVICES=dynamodb -e DEBUG=1 --rm localstack/localstack:0.10.5"
```

Comment:
```suggestion
    "localstack": "docker run -it -p 4566:4566 -e SERVICES=dynamodb -e DEBUG=1 --rm localstack/localstack"
```

Reply:
We should use a specific version of localstack. Never the latest. I had very bad experience using the latest one.

Reply:
Localstack is well know for breaking things on new releases. It happened to me twice in the past. version `0.10.5` is pretty stable and it's what we use in kubernetes-external-secrets.

Reply:
I think we should take the latest one as of now, pin to that minor version (e.g. 0.11) and update as needed later.

Reply:
Done.